packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -1432,111 +1240,82 @@ export namespace EmrCreateCluster {
     /**
      * The identifier of the Amazon EC2 security group for the master node.
      *
-     * @default No emrManagedMasterSecurityGroup
+     * @default - None
      */
     readonly emrManagedMasterSecurityGroup?: string;
 
     /**
      * The identifier of the Amazon EC2 security group for the core and task nodes.
      *
-     * @default No emrManagedSlaveSecurityGroup
+     * @default - None
      */
     readonly emrManagedSlaveSecurityGroup?: string;
 
     /**
      * Applies only to Amazon EMR release versions earlier than 4.0. The Hadoop version for the cluster.
      *
-     * @default No hadoopVersion
+     * @default - None
      */
     readonly hadoopVersion?: string;
 
     /**
      * The number of EC2 instances in the cluster.
      *
-     * @default No instanceCount
+     * @default - None
      */
     readonly instanceCount?: number;
 
     /**
      * Describes the EC2 instances and instance configurations for clusters that use the instance fleet configuration.
      *
-     * @default No instanceFleets
+     * @default - None
      */
     readonly instanceFleets?: InstanceFleetConfigProperty[];
 
     /**
      * Configuration for the instance groups in a cluster.
      *
-     * @default No instanceGroups
+     * @default - None
      */
     readonly instanceGroups?: InstanceGroupConfigProperty[];
 
     /**
      * The EC2 instance type of the master node.
      *
-     * @default No masterInstanceType
+     * @default - None
      */
     readonly masterInstanceType?: string;
 
     /**
      * The Availability Zone in which the cluster runs.
      *
-     * @default EMR selected default
+     * @default - EMR selected default
      */
     readonly placement?: PlacementTypeProperty;
 
     /**
      * The identifier of the Amazon EC2 security group for the Amazon EMR service to access clusters in VPC private subnets.
      *
-     * @default No serviceAccessSecurityGroup
+     * @default - None
      */
     readonly serviceAccessSecurityGroup?: string;
 
     /**
      * The EC2 instance type of the core and task nodes.
      *
-     * @default No slaveInstanceThpe
+     * @default - None
      */
     readonly slaveInstanceType?: string;
 
     /**
      * Specifies whether to lock the cluster to prevent the Amazon EC2 instances from being terminated by API call, user intervention,
      * or in the event of a job-flow error.
      *
-     * @default EMR selected default (false)
+     * @default - None
```

Comment:
Make me no sense. Default is either true or false, can't be none.

Reply:
changed it to true